src/main/java/filemanager/resource/ClientResource.java

```diff
@@ -19,16 +19,20 @@
 
     private FeedExporter feedExporter;
 
+    private RequestPassCheck check;
+
     @Inject
-    public ClientResource(FeedExporter feedExporter) {
+    public ClientResource(FeedExporter feedExporter, RequestPassCheck check) {
```

Comment:
Nope, please use Guice abilities to intercept method calls. There should be no changes to ```ClientResource``` class.